package.json

```diff
@@ -17,15 +17,15 @@
   "dependencies": {
     "isomorphic-qwerty": "^0.0.2",
     "jszip": "^3.10.1",
-    "moment-of-symmetry": "^0.4.0",
+    "moment-of-symmetry": "github:xenharmonic-devs/moment-of-symmetry",
```

Comment:
Only review. No merge please. I will publish this package first if everything is OK.